tests/mixing/test_mix.py

```diff
@@ -0,0 +1,88 @@
+import logging, traceback
+import gc, shutil
+import torch
+
```

Comment:
pytest will attempt to run test_* files during CI, but looks like this is just being run as a script. Can this be reformulated as a pytest test?

Reply:
done